internal/services/kusto/kusto_eventgrid_data_connection_resource_test.go

```diff
@@ -81,7 +82,8 @@ func TestAccKustoEventGridDataConnection_mappingRule(t *testing.T) {
 func TestAccKustoEventGridDataConnection_userAssignedIdentity(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_kusto_eventgrid_data_connection", "test")
 	r := KustoEventGridDataConnectionResource{}
-
+	fmt.Println(r.userAssignedIdentity(data))
+	t.Skip()
```

Comment:
Looks like these are some debug helpers that were leftover by mistake?

Reply:
Ah indeed, thanks! I need to stop doing that.